cmd/goodman/main.go

```diff
@@ -51,7 +51,7 @@ func main() {
 				}
 			}()
 			for retries := 5; retries > 0; retries-- {
-				runner, err := goodman.NewRunner("Hooks", hookServerInitalPort)
+				runner, err := goodman.NewRunner("HooksRunner", hookServerInitalPort)
```

Comment:
That would look like this:
```
rpcService := reflect.TypeOf((*hooks.HooksRunner)(nil)).Elem().Name()
for retries := 5; retries > 0; retries-- {
	runner, err := goodman.NewRunner(rpcService, hookServerInitalPort)
```
and needs an import of `github.com/snikch/goodman/hooks`
Do you prefer that?